packages/ccip-svm/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "lib": ["es2020", "esnext"],
+    "target": "ES2020",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "outDir": "./dist",
+    "rootDir": "./src",
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
```

Comment:
ive not worked with Solana Kit much - lets keep an eye on skipLibCheck in case that dep is not consistent wth typing?